### PR TITLE
chore: clean and update dependencies

### PR DESCRIPTION
**Pull request by afdesk**
I did next actions:
1. remove `go.sum` file,
2. remove `require` part in go.mod manually,
3. run `go mod tidy`

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,39 +4,37 @@ go 1.16
...
 require (
 	github.com/BurntSushi/toml v0.4.1
-	github.com/GoogleCloudPlatform/docker-credential-gcr v1.5.0
-	github.com/alicebob/miniredis/v2 v2.16.0
-	github.com/aquasecurity/cfsec v0.2.2
-	github.com/aquasecurity/defsec v0.0.37
+	github.com/GoogleCloudPlatform/docker-credential-gcr v2.0.5+incompatible
+	github.com/alicebob/miniredis/v2 v2.16.1
+	github.com/aquasecurity/cfsec v0.3.1
+	github.com/aquasecurity/defsec v0.1.1
```
Comment (knqyf263) [inline go.mod:10]: I don't want to update these versios until @owenrumney allows it.

File: go.mod [hunk 2]
```diff
@@ -4,39 +4,37 @@ go 1.16
...
 	github.com/aquasecurity/go-dep-parser v0.0.0-20211110174639-8257534ffed3
 	github.com/aquasecurity/testdocker v0.0.0-20210911155206-e1e85f5a1516
-	github.com/aquasecurity/tfsec v0.61.4-0.20211130135821-0a1997e494fa
-	github.com/aws/aws-sdk-go v1.42.0
-	github.com/docker/docker v20.10.10+incompatible
+	github.com/aquasecurity/tfsec v0.61.3
```
Comment (knqyf263) [inline go.mod:13]: ditto
https://github.com/aquasecurity/fanal/pull/324

File: go.mod [hunk 3]
```diff
@@ -4,39 +4,37 @@ go 1.16
...
-	github.com/opencontainers/image-spec v1.0.2-0.20190823105129-775207bd45b6
-	github.com/saracen/walker v0.0.0-20191201085201-324a081bae7e
+	github.com/opencontainers/image-spec v1.0.2
+	github.com/saracen/walker v0.1.2
 	github.com/sosedoff/gitkit v0.3.0
 	github.com/stretchr/testify v1.7.0
-	github.com/testcontainers/testcontainers-go v0.11.1
-	github.com/tmccombs/hcl2json v0.3.1
+	github.com/testcontainers/testcontainers-go v0.12.0
+	github.com/tmccombs/hcl2json v0.3.3
```
Comment (knqyf263) [inline go.mod:37]: AFAIK, this update breaks tests.
https://github.com/aquasecurity/fanal/pull/242

File: go.mod [hunk 4]
```diff
@@ -4,39 +4,37 @@ go 1.16
...
 	github.com/knqyf263/go-apk-version v0.0.0-20200609155635-041fdbb8563f
 	github.com/knqyf263/go-deb-version v0.0.0-20190517075300-09fca494f03d
 	github.com/knqyf263/go-rpmdb v0.0.0-20210911072402-73bd0ce46c49
 	github.com/knqyf263/nested v0.0.1
 	github.com/kylelemons/godebug v1.1.0
-	github.com/mitchellh/mapstructure v1.4.2
-	github.com/moby/buildkit v0.8.1
-	github.com/open-policy-agent/opa v0.34.0
+	github.com/mitchellh/mapstructure v1.4.3
+	github.com/moby/buildkit v0.9.3
```
Comment (knqyf263) [inline go.mod:29]: ditto

**Comment by afdesk** [comment]
this PR requires too much changes.

**Pull request closed without merging**